cmd/statusd-cli/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+var (
+	statusdConn = flag.String("statusd", "localhost", "set host of statusd")
+)
+
+// main is the entrypoint for the statusd command line interface.
+func main() {
+	flag.Usage = printUsage
+	flag.Parse()
+
+	fmt.Printf("statusd-cli connecting statusd on %v\n", *statusdConn)
+
+	// Starting REPL.
+	repl, err := NewREPL("localhost")
```

Comment:
It should be `NewREPL(*statusdConn)` I suppose.

Reply:
Eh, it should. 🙄